### PR TITLE
Feature/jude/create notifications

### PR DESCRIPTION
**Pull request by judeokafor**
User creation, user suspended, location created, new commercial

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -28,3 +29,73 @@ app.use((err, req, res, next) => {
...
 });
 exports.api = functions.region('europe-west1').https.onRequest(app);
+exports.signUpEmailNotification = functions.region('europe-west1')
+    .firestore.document('users/{userId}').onCreate(async snapshot => {
+        try {
+            const { email, firstName, lastName, userId } = snapshot.data();
+            const receipent = email;
+            const subject = 'Welcome to NeoNatar';
+            const text = `Dear ${firstName} ${lastName},
+        reach out to all your esteemed customers suing our platform`;
```
Comment (codemutatio) [inline functions/index.js:39]: I think the suing should be using

Comment (enyeInc) [reply functions/index.js:39]: why are these strings here?

File: functions/src/controllers/notifications.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	 * @param {object} res - response object
+	 * @return  {Object} result
+	 */
+    static async allNotifications(req, res) {
+        try {
+            const { userId } = req.user;
+            const data = await db.collection('notifications')
+                .where('userId', '==', userId).where('read', '==', false)
+                .orderBy('createdAt', 'desc').get();
+            const docs = data.docs, notifications = [];
```
Comment (enyeInc) [inline functions/src/controllers/notifications.js:21]: don't do this, put notifications on its own line with its own `const`

File: functions/src/controllers/notifications.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            for (const doc of docs) {
+                notifications.push({ id: doc.id, notification: doc.data() });
+            }
```
Comment (enyeInc) [inline functions/src/controllers/notifications.js:24]: One line
```
const notifications = docs.map(doc => ({ id: doc.id, notification: doc.data() }));
```

**Review by enyeInc — APPROVED** [review]
Make the minor changes, LGTM

File: functions/index.js (hunk 1 above)
Comment (judeokafor) [reply functions/index.js:39]: It is a temporary text sent as an email notification. Pending when we design a better UI page for email notification.

File: functions/src/controllers/notifications.js (hunk 2 above)
Comment (judeokafor) [reply functions/src/controllers/notifications.js:21]: Okay.

File: functions/src/controllers/notifications.js (hunk 3 above)
Comment (judeokafor) [reply functions/src/controllers/notifications.js:24]: okay

**Pull request merged**